src/plots/rectanglePlot.ts

```diff
@@ -251,6 +253,50 @@ export module Plots {
       });
     }
 
+    /**
+     * Gets the Accessor for label.
+     */
+    public label(): AccessorScaleBinding<any, string>;
```

Comment:
Just restating when we need to make a property; if we have a need to port data values through a scale to make them visually presentable, then we should make that aspect a property.